extensions/aproc/proc/ingest/drivers/impl/cosmoskymed.py

```diff
@@ -146,34 +155,40 @@ def to_item(self, url: str, assets: list[Asset]) -> Item:
             ),
             assets=dict(map(lambda asset: (asset.name, asset), assets))
         )
+
+        if tif_path != self.tif_path:
+            os.remove(tif_path)
+        if met_path != self.met_path:
+            os.remove(met_path)
```

Comment:
The system should never delete any file. If it is really needed, then this must be documented and we must use the comment "!DELETE!".

https://docs.google.com/document/d/1UTm5JUZ8F9ptlcya9v88hzg-iPWxeuCCziAD_oycloo/edit?tab=t.0